dev/run

```diff
@@ -504,6 +512,119 @@ def boot_nouveau(ctx):
     )
 
 
+CLOUSEAU_DIR = "clouseau"
+JAVA_VERSION_RE = re.compile('"(\d+\.\d+).*"')
+
+
+def get_java_version(java):
+    output = sp.check_output([java, "-version"], stderr=sp.STDOUT).decode("utf-8")
+    if output:
+        matches = JAVA_VERSION_RE.search(output)
+        if matches:
+            return float(matches.groups()[0])
+
+
+@log("Start Clouseau node clouseau{idx}")
+def boot_clouseau(ctx, idx):
+    def warn(reason):
+        print("Warning: " + reason)
```

Comment:
Instead of multiple `return` in the following if statements, shall we add `sys.exit()` after this line?

- If we use `return` the log looks like this and never stops until we terminate it manually.
```bash
[ * ] Setup environment ... ok
[ * ] Ensure CouchDB is built ... ok
[ * ] Ensure Erlang boot script exists ... ok
[ * ] Prepare configuration files ... ok
[ * ] Start node node1 ... ok
[ * ] Start Clouseau node clouseau1 ... Warning: Clouseau deployment cannot be found, please run `./configure with --enable-clouseau`
ok
[ * ] Check node at http://127.0.0.1:15984/ ... ok
[ * ] Check Clouseau node clouseau1 ... failed: Clouseau is not working
[ * ] Check node at http://127.0.0.1:15984/ ... ok
[ * ] Check Clouseau node clouseau1 ... failed: Clouseau is not working
[ * ] Check node at http://127.0.0.1:15984/ ... ok
...
```
- If we use `sys.exit()` it will stop immediately after the warning.
```bash
[ * ] Setup environment ... ok
[ * ] Ensure CouchDB is built ... ok
[ * ] Ensure Erlang boot script exists ... ok
[ * ] Prepare configuration files ... ok
[ * ] Start node node1 ... ok
[ * ] Start Clouseau node clouseau1 ...
Warning: Clouseau deployment cannot be found, please run `./configure --enable-clouseau`
```

Reply:
The "check node" action is not repeated indefinitely but for a fixed amount (10) of times.  See the `ensure_all_nodes_alive()` function for the logic.  I admit that it could be annoying but the script will eventually give up on trying.

Perhaps it may make sense to differentiate between warnings and errors.  In case of errors, the script could stop earlier ("fail fast").  I am not convinced that `sys.exit()` is the best to express that though.  I consider it an aggressive break in the control flow and I usually keep it as the last resort.  I will look into options on how to implement this.